website/api/zkvm/precompiles.md

```diff
@@ -1,22 +1,23 @@
-# Cryptography Acceleration
+# Precompiles
 
 RISC Zero's rv32im implementation includes a number of specialized extension
-circuits, including two "accelerators" for cryptographic functions: SHA-256 and
-[256-bit modular multiplication][bigint], referred to as "bigint"
-multiplication. By implementing these operations directly in the "hardware" of
-the zkVM, programs that use these accelerators execute faster and can be proven
+circuits, including "precompiles" for cryptographic and algebraic functions: SHA-256,
+Keccak, RSA, elliptic curve, and modular multiplication operations.
+By implementing these operations directly in the "hardware" of
+the zkVM, programs that use these precompiles execute faster and can be proven
 with significantly less resources [^1].
 
 ## Accelerated Crates
 
-The SHA-256 and bigint accelerators are currently integrated in "accelerated"
+Our precompiles are currently integrated in "accelerated"
```

Comment:
mb we want to reference the term "patched crates" here? not strongly opinionated.